File: app/src/main/java/com/xjx/helper/test/TestActivity.java
package com.xjx.helper.test;

import android.view.View;

import com.xjx.helper.R;
import com.xjx.helper.base.CommonBaseTitleActivity;
import com.xjx.helper.enums.PlaceholderStatus;
import com.xjx.helper.utils.LogUtil;

/**
 * 测试页面
 */
public class TestActivity extends CommonBaseTitleActivity {

    private android.widget.Button mButton;
    private android.widget.EditText mEt1;
    private android.widget.EditText mEt2;
    private android.widget.TextView mTv1;
    private android.widget.TextView mTv2;

    @Override
    protected int getTitleLayout() {
        return R.layout.activity_test;
    }

    @Override
    protected void initView() {
        super.initView();

        mButton = findViewById(R.id.button);
        mEt1 = findViewById(R.id.et_1);
        mEt2 = findViewById(R.id.et_2);
        mTv1 = findViewById(R.id.tv1);
        mTv2 = findViewById(R.id.tv2);
    }

    @Override
    protected void initData() {
        super.initData();
        SwitchLoadingStatus(PlaceholderStatus.NONE);

        String abc = "123";
        if (abc != null) {
            LogUtil.e("不为空");
        }
        if (abc.equals("123")) {
            LogUtil.e("123");
        } else {
            LogUtil.e("othier");
        }

        mButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                String value1 = mEt1.getText().toString();
                String value2 = mEt2.getText().toString();

                int parseInt1 = Integer.parseInt(value1);
                int parseInt2 = Integer.parseInt(value2);

                int i = parseInt1 / parseInt2;

                mTv1.setText("" + i);

                int i1 = parseInt1 % parseInt2;

                mTv2.setText("" + i1);
            }
        });

    }
}

File: app/src/main/java/com/xjx/helper/ui/home/activity/TestActivity.kt
package com.xjx.helper.ui.home.activity

import android.view.View
import com.xjx.helper.base.CommonBaseTitleActivity
import com.xjx.helper.enums.PlaceholderStatus
import kotlinx.android.synthetic.main.activity_test.*
import java.text.SimpleDateFormat
import java.util.*


class TestActivity : CommonBaseTitleActivity() {

    override fun getTitleLayout(): Int {
        return com.xjx.helper.R.layout.activity_test
    }

    override fun initListener() {
        super.initListener()
        button.setOnClickListener(this)
    }

    override fun initData() {
        super.initData()

        SwitchLoadingStatus(PlaceholderStatus.NONE)

    }

    override fun onClick(v: View?) {
        super.onClick(v)
        when (v?.id) {
            com.xjx.helper.R.id.button -> {

                val current = Calendar.getInstance()
                current.time = Date()//填入当前时间

                //当前年
                val year = current.get(Calendar.YEAR)
                //当前月
                val month = current.get(Calendar.MONTH)
                //当前月的第几天：即当前日
                val day_of_month = current.get(Calendar.DAY_OF_MONTH)
                //当前时：HOUR_OF_DAY-24小时制；HOUR-12小时制
                val hour = current.get(Calendar.HOUR_OF_DAY)
                //当前分
                val minute = current.get(Calendar.MINUTE)

                // 开始时间
                val start = Calendar.getInstance()
                start.set(year, month, day_of_month + 1, hour, minute)

                val endDate = Calendar.getInstance()
                endDate.set(year, month, day_of_month + 10, hour, minute)

                val s = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")

                tv1.text = "当前时间：" + start.get(Calendar.YEAR) + "-" + (start.get(Calendar.MONTH) + 1) + "-" + start.get(Calendar.DAY_OF_MONTH) + " " + start.get(Calendar.HOUR_OF_DAY) + ":" + start.get(Calendar.MINUTE)
                tv2.text = "当前时间：" + endDate.get(Calendar.YEAR) + "-" + (endDate.get(Calendar.MONTH) + 1) + "-" + endDate.get(Calendar.DAY_OF_MONTH) + " " + endDate.get(Calendar.HOUR_OF_DAY) + ":" + endDate.get(Calendar.MINUTE)


            }
        }
    }


}
